### PR TITLE
Enables enter key usage for checkboxes

### PR DESCRIPTION
**Pull request by Kristyn-Mimura**
# Ticket Link

[Ticket](https://uhawaii.atlassian.net/browse/GROUPINGS-1383)

# List of squashed commits
- Implements enter key functionality to exclude.html & include.html
- renames createSyncDestModal
- Uses html for enter key usage for remaining checkboxes
- Updates to use angularJS for all enter-key functionality for checkboxes 

# Test Checklist

- [X] Unit Tests Passed:
- [X] Jasmine Tests Passed:
- [X] General Visual Inspection:

### CODE REVIEW

**Comment by JorWo** [comment]
For all places you used onkeydown, could you use a ternary operator like this: 
`onkeydown="event.keyCode === 13 ? this.click() : null"`. 
This is to match the other places in the HTML that use event.keyCode === 13. 

Also, could you add using the enter key to the rest of the checkboxes such as in preferences as well as the checkboxes for every member in include/exclude?

One more thing, although it is not in the scope of the ticket, we can fix it now. I noticed in sync-destinations.html that we forgot to rename createSyncDestModal to displaySyncDestModal. So currently our sync destinations checkbox is not working properly. Thanks!

**Comment by JorWo** [comment]
What's the status of this PR?

File: src/main/resources/static/javascript/mainApp/general.controller.js [hunk 1]
```diff
@@ -1777,7 +1777,8 @@
         };
 
         $scope.updateIncludeCheck = function () {
-            $scope.includeCheck = $scope.includeCheck === false;
+            //$scope.includeCheck = $scope.includeCheck === false;
```
Comment (JorWo) [inline src/main/resources/static/javascript/mainApp/general.controller.js:1780]: There is a leftover comment here.

File: src/main/resources/static/javascript/mainApp/general.controller.js [hunk 2]
```diff
@@ -2099,6 +2099,18 @@
             }
             return "/uhgroupings";
         };
+
+        $scope.clickCheckBox = function (i) {
+            $scope.membersInCheckboxList[i.uhUuid] = !$scope.membersInCheckboxList[i.uhUuid];
+        };
```
Comment (JorWo) [inline src/main/resources/static/javascript/mainApp/general.controller.js:2105]: Can you move the `if (event.keyCode === 13 )` condition into this function just like you did with `$scope.clickWithEnter()`? Then in places in HTML that have: `ng-keydown="$event.keyCode === 13 ? clickCheckBox(i): null"`, you can get rid of `$event.keyCode === 13`.

Comment (Kristyn-Mimura) [reply src/main/resources/static/javascript/mainApp/general.controller.js]: I made the changes.

**Review by JorWo — APPROVED** [review]

**Pull request merged**